README.md

```diff
@@ -22,11 +22,11 @@ See [https://docs.msupply.foundation/en:cold_chain:start](https://docs.msupply.f
 - Install Android Studio and SDK tools: https://developer.android.com/studio.
 - Install SDKMAN for managing Java versions: https://sdkman.io/.
 
-The application is using java v1.8 you may need to target a specific java version in order to run the app locally.
-e.g.
+The application is using java v11 you may need to target a specific java version in order to run the app locally.
+e.g. (with sdkman installed):
 
 ```
-export JDK_HOME=~/.jenv/versions/oracle64-1.8.0.192 && JAVA_HOME=~/.jenv/versions/oracle64-1.8.0.192 && yarn start
+ sdk use java 11.0.23-zulu
```

Comment:
Added this as it seems we need java11+ now that we're targeting a SDK 23.
Haven't tired 17 or other more modern ones, wasn't working before...

Reply:
api level 23 is [android 6](https://developer.android.com/tools/releases/platforms#6.0) - given that the samsung I'm using is older, and android 12.. we're probably ok